### PR TITLE
Fixes #48 - new subfolder hierarchy

### LINKED ISSUE #48: Rearrange shadow plugin sub-menus
@jlaforga points out that Change Controllers plugin is a bit confusing for the user, whereas Teach Mode plugin is easier.

We probably should create a `basic` and an `advanced` submenus under Shadow Robot.

To do this is enough to add a group tag in the `..._plugin.xml` e.g:

https://github.com/shadow-robot/sr-visualization/blob/indigo-devel/sr_gui_teach_mode/sr_teach_mode_plugin.xml

### PR DESCRIPTION
**Pull request by ugocupcic**
Lots of moving around but no major changes

### CODE REVIEW

**Comment by ugocupcic** [comment]
image

image

File: advanced/sr_gui_cyberglove_calibrator/sr_cyberglove_calibrator_plugin.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<library path="src">
+  <class name="Cyberglove_Calibrator"
+         type="sr_gui_cyberglove_calibrator.cyberglove_calibrator.SrGuiCybergloveCalibrator"
+         base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A plugin for .
```
Comment (kirstyellis) [inline advanced/sr_gui_cyberglove_calibrator/sr_cyberglove_calibrator_plugin.xml]: This description is a bit brief...

File: advanced/sr_gui_cyberglove_calibrator/sr_cyberglove_calibrator_plugin.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <qtgui>
+      <group>
+        <label>Shadow Robot</label>
+      </group>
+      <group>
+        <label>Advanced</label>
+      </group>
+      <icon type="file">../../sr_visualization_icons/icons/hand.png</icon>
+      <label>Glove calibrator</label>
+      <statustip>A plugin for .</statustip>
```
Comment (kirstyellis) [inline advanced/sr_gui_cyberglove_calibrator/sr_cyberglove_calibrator_plugin.xml]: here also

File: advanced/sr_gui_movement_recorder/sr_gui_movement_recorder_plugin.xml [hunk 3]
```diff
@@ -1,13 +1,17 @@
 <library path="src">
-  <class name="SrGuiMovementRecorder" type="sr_gui_movement_recorder.movement_recorder.SrGuiMovementRecorder" base_class_type="rqt_gui_py::Plugin">
+  <class name="SrGuiMovementRecorder" type="sr_gui_movement_recorder.movement_recorder.SrGuiMovementRecorder"
+         base_class_type="rqt_gui_py::Plugin">
     <description>
-	A plugin for playing and saving movements.
+      A plugin for playing and saving movements.
```
Comment (kirstyellis) [inline advanced/sr_gui_movement_recorder/sr_gui_movement_recorder_plugin.xml:5]: the statustip says the plugin is for loading the different controllers...

**Pull request merged**